SwiftLSPClient/Types/Basic.swift

```diff
@@ -52,6 +52,7 @@ public enum TextDocumentSyncKind: Int, Codable {
 public enum LanguageIdentifier: String, Codable, CaseIterable {
     case go = "go"
     case json = "json"
+    case swift = "swift"
```

Comment:
I'm mixed up - wasn't this change already merged?

Reply:
It was, I bungled the branch, shouldn't matter. I've added a few extra types in new commit.